lldb/test/API/lang/swift/async/stepping/step-in/task-switch/TestSwiftTaskSwitch.py

```diff
@@ -0,0 +1,35 @@
+import lldb
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+
+
+class TestCase(lldbtest.TestBase):
+    @swiftTest
+    @skipIf(oslist=["windows", "linux"])
+    def test(self):
+        """Test conditions for async step-in."""
+        self.build()
+
+        src = lldb.SBFileSpec("main.swift")
+        target, _, thread, _ = lldbutil.run_to_source_breakpoint(self, "await f()", src)
+        self.assertEqual(thread.frame[0].function.mangled, "$s1a5entryO4mainyyYaFZ")
+
+        function = target.FindFunctions("$s1a5entryO4mainyyYaFZTQ0_")[0].function
```

Comment:
can you assert that function is non-null? Will be easier to debug that way.

Reply:
https://github.com/apple/llvm-project/pull/6908